modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -52,133 +86,174 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
+**PR:** [#2091](https://github.com/Workday/canvas-kit/pull/2091)
+
+In v7, we released an update to our depth tokens that was too bold and harsh for web applications.
+We've modified the depth token styles to be more subtle and improve visual design. This change
+affects all components which use depth tokens including `Card`, `Toast`, `Dialog`, `Popup`, `Modal`,
+and `Menu`. We have not changed which depth values each component references (i.e., `Card` remains
+at `depth[1]`).
 
-#### Soft Deprecation
+## Component Removals
 
-A soft-deprecated component is still available with its full functionality, but it will have been
-renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
-announcing its deprecation. This warning will only be triggered on the component's initial render.
+### Drawer
 
-Soft-deprecated types and utilities will also have been renamed but generally will not trigger a
-console warning.
+**PR:** [#1970](https://github.com/Workday/canvas-kit/pull/1970)
 
-#### Hard Deprecation
+We've removed the `Drawer` component from Canvas Kit (for reference, see the
+[`Drawer`](https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/labs-drawer--basic) from v8). Please
+use the [`SidePanel`](/components/containers/side-panel/) in Preview instead.
 
-A hard-deprecated component or package is no longer available. You will need to follow the method
-prescribed in our Upgrade Guide to update your application. Please
-[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) to our team
-directly if you need additional help.
+> **Note:** The `SidePanel` in Main will eventually be replaced with the `SidePanel` in Preview. We
+> recommend you use the `SidePanel` in Preview until then.
 
 ---
 
-### Drawer
+### Layout and Column
+
+**PR:** [#2018](https://github.com/Workday/canvas-kit/pull/2018)
 
-<!---
-TODO: Please review this section with documenting SidePanel deprication
--->
+We've removed the `Layout` and `Column` components from Canvas Kit (for reference, see
+[`Column and Layout`](https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/components-containers-column-and-layout--12-column-layout)
+from v8). Please use [`Grid`](/components/layout/grid/) instead. While `Grid` is not a 1:1
+replacement for `Layout` and `Column`, it can be used to generate the same types of layouts and
+offers a more robust and flexible layout solution.
 
-We've [hard-deprecated](#hard-deprecation) `Drawer` and this component has been fully removed from
-Canvas Kit. Consider using the
-[Side Panel](https://workday.github.io/canvas-kit/?path=/story/preview-side-panel--basic). Be Aware
-that `SidePanel` from the Main package is planned to be replaced by the one from Preview. We suggest
-you use the Preview version of `SidePanel` until the one in Main is replaced with it.
+Please refer to our [Layout examples](/examples/layout) for examples of how to implement common
+layouts using `Grid`.
```

Comment:
This section originally read:

> Please refer to the [Examples section](https://workday.github.io/canvas-kit/?path=/docs/examples-layouts--custom-column-positioning) for ideas on how to replace the deprecated `Layout` component with the `Grid` component.

That suggested the examples would start with code using `Layout` and show you how to implement the same layout using `Grid`. That's not quite what these examples are however (there's no "before" with `Layout`) so I changed this section to "examples of how to implement common layouts."